dvc/stage.py

```diff
@@ -772,6 +773,28 @@ def _run(self):
         if p.returncode != 0:
             raise StageCmdFailedError(self)
 
+    def _import(self, dry, resume, force):
+        logger.info(
+            "Importing '{dep}' -> '{out}'".format(
+                dep=self.deps[0], out=self.outs[0]
+            )
+        )
+
+        if dry:
+            return
+
+        if self._already_cached() and not force:
+            self.outs[0].checkout()
+            return
+
+        if not hasattr(self.deps[0], "pkg"):
+            self.deps[0].download(self.outs[0].path_info, resume=resume)
```

Comment:
can we actually provide an empty download for the pkg dependency and simplify the code, remove this if

Reply:
@shcheklein Good point. I've adjusted the `download` method to hide the logic it under it, so it looks much better.